tests/test_spstats.py

```diff
@@ -45,6 +45,29 @@ def load_diff() -> tuple[gu.georaster.Raster, np.ndarray]:
 
     return diff, mask
 
+def load_ref_and_diff() -> tuple[gu.georaster.Raster, gu.georaster.Raster, np.ndarray]:
```

Comment:
So far, we've always put the test data in the definition of the class, see e.g. https://github.com/GlacioHack/xdem/blob/2a389a09ae4d68f7e765e576b1c9afd99021dc92/tests/test_volume.py#L13
I don't know which one is better. Maybe @erikmannerfelt has an idea?
I guess it makes sense to have a function that loads this sample data if often used... On the other hand, it's weird to have a function in the tests that is never tested! ^_^

Reply:
Actuallty @erikmannerfelt added this a couple months ago, so should be OK :wink: 
I will collapse the two functions into one though, it will save space

Reply:
Done in 3fd6d83